kalliope/core/RestAPI/FlaskAPI.py

```diff
@@ -22,11 +25,20 @@ def __init__(self, app, port=5000, brain=None):
         self.app = app
         self.port = port
         self.brain = brain
+        self.allow_cors_request = allow_cors_request
 
         # Flask configuration remove default Flask behaviour to encode to ASCII
         self.app.url_map.strict_slashes = False
         self.app.config['JSON_AS_ASCII'] = False
 
+        if self.allow_cors_request is True:
+            try:
+              from flask_cors import CORS, cross_origin
```

Comment:
this is natives to Flask? If not, it should be added to the dependencies list into the setup.py. (And in the requirement.txt used by the manual install)